designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Examples.scala

```diff
@@ -0,0 +1,227 @@
+
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.ui.api.description.scenarioActivity.Dtos.ScenarioActivityError.{NoComment, NoScenario}
+import pl.touk.nussknacker.ui.api.description.scenarioActivity.Dtos.{
+  Attachment,
+  Comment,
+  FoundActivity,
+  ScenarioActivities,
+  ScenarioActivitiesSearchResult,
+  ScenarioActivity,
+  ScenarioAttachments,
+  ScenarioCommentsAndAttachments
+}
+import sttp.tapir.EndpointIO.Example
+
+import java.time.Instant
+import java.util.UUID
+
+object Examples {
+
+  val scenarioCommentsAndAttachments: Example[ScenarioCommentsAndAttachments] = Example.of(
+    summary = Some("Display scenario activity"),
+    value = ScenarioCommentsAndAttachments(
+      comments = List(
+        Comment(
+          id = 1L,
+          scenarioVersion = 1L,
+          content = "some comment",
+          user = "test",
+          createDate = Instant.parse("2024-01-17T14:21:17Z")
+        )
+      ),
+      attachments = List(
+        Attachment(
+          id = 1L,
+          scenarioVersion = 1L,
+          fileName = "some_file.txt",
+          user = "test",
+          createDate = Instant.parse("2024-01-17T14:21:17Z")
+        )
+      )
+    )
+  )
+
+  val scenarioActivities: Example[ScenarioActivities] = Example.of(
+    summary = Some("Display scenario actions"),
+    value = ScenarioActivities(
+      activities = List(
+        ScenarioActivity.forScenarioCreated(
+          id = UUID.randomUUID().toString,
+          user = "some user",
+          date = Instant.parse("2024-01-17T14:21:17Z"),
+          scenarioVersion = 1,
+          comment = None,
+        ),
+        ScenarioActivity.forScenarioArchived(
+          id = UUID.randomUUID().toString,
+          user = "some user",
+          date = Instant.parse("2024-01-17T14:21:17Z"),
+          scenarioVersion = 1,
+          comment = None,
+        ),
+        ScenarioActivity.forScenarioUnarchived(
+          id = UUID.randomUUID().toString,
+          user = "some user",
+          date = Instant.parse("2024-01-17T14:21:17Z"),
+          scenarioVersion = 1,
+          comment = None,
+        ),
+        ScenarioActivity.forScenarioDeployed(
+          id = UUID.randomUUID().toString,
+          user = "some user",
+          date = Instant.parse("2024-01-17T14:21:17Z"),
+          scenarioVersion = 1,
+          comment = Some("Deployment of scenario - task JIRA-1234"),
+        ),
+        ScenarioActivity.forScenarioCanceled(
+          id = UUID.randomUUID().toString,
+          user = "some user",
+          date = Instant.parse("2024-01-17T14:21:17Z"),
+          scenarioVersion = 1,
+          comment = Some("Canceled because marketing campaign ended"),
+        ),
+        ScenarioActivity.forScenarioModified(
+          id = UUID.randomUUID().toString,
+          user = "some user",
+          date = Instant.parse("2024-01-17T14:21:17Z"),
+          scenarioVersion = 1,
```

Comment:
Based on mocks, there should be no scenario version in this case, so maybe the backend won't send it?

![image](https://github.com/user-attachments/assets/f16e09a3-8283-4846-9d16-4742e0979414)

Reply:
After implementing most of BE, I encountered this issue too, it will be optional field.